recipes/recipes/emscripten_emscripten-wasm32/recipe.yaml

```
@@ -23,6 +23,7 @@ outputs:
       build:
         - curl
         - python
+        - gettext
```

I hit a similar problem on a clean Ubuntu docker image, and
```suggestion
        - gettext-tools
```
solved it (brings less deps than `gettext`)

Also had to add this dep to `recipes/recipes/cross-python_emscripten-wasm32/recipe.yaml`, which uses `envsubst` on activation script